src/System.ServiceModel.Syndication/ref/System.ServiceModel.Syndication.netcoreapp.cs

```diff
@@ -7,8 +7,26 @@
 
 namespace System.ServiceModel.Syndication
 {
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
```

Comment:
Why explicitly sequential layout on structs?

Reply:
I think this is a hangover from how we used to create contracts with tooling and it made everything explicit. This wasn't my doing, but looking through the rest of the repo these aren't being used so I'll remove them.

Reply:
If it is just the tool doing that, then leave it be. If it comes from original source, I think we should have a reason.

Reply:
Yeah, I see the source doesn't have it - I forgot to check it earlier (I meant to, then got distracted).
All is good from my side. Tests are passing. get a code review of your delta and feel free to merge.

Reply:
A tool didn't create this instance. I believe this was put in because previously tooling put that in and it was being copied. But I checked and that's not the current convention.

Reply:
I see, then let's get rid of it :)